tests/data/circuit_config.json

```diff
@@ -16,6 +16,9 @@
         "populations": {
             "default": {
                 "type": "biophysical"
+            },
+            "default2": {
```

Comment:
`default2` does not really make sense to me as a name, how does one use two defaults? 🙂

Reply:
Yeah, `default` and `default2` aren't meaningful names, and they don't follow the new naming convention described at https://sonata-extension.readthedocs.io/en/latest/sonata_population.html.

However, this is only for test so the naming is less important.

The names `default` and `default2` are the population names present in the h5 files already used for testing.
You can see their content opening the files with h5py, or with the cli for example with:
```
h5ls -r tests/data/nodes.h5
h5ls -r tests/data/edges.h5
```